signup.php

```diff
@@ -37,6 +37,7 @@
     $_REQUEST['postal'],
     $_REQUEST['country']);
 $lead->setRating('N');
+$lead->setSendSalesRepAssignmentNotification(true);
```

Comment:
maybe this should just be sendAssignmentNotification()

inside it will set the value, but there would be no unset, to unset you would just not set it.

are we falling into the boolean trap?